katalogus/templates/clone_settings.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
```

Comment:
Should this extend a base layout or is it used as a partial template?

Reply:
Used as partial as far as I can see